Data/ShippingConfirmations.cs

```diff
@@ -87,13 +92,14 @@ FROM shipping_confirmation
         }
 
         /// <summary>
-        /// Inserts a shipping confirmation into the database. All operations around this method are surrounded by a transaction which rollbacked if an exception is encountered, commited otherwise. If an exception is encountered, it is re-thrown.
+        /// Inserts a shipping confirmation into the database.
         /// </summary>
         /// <param name="shippingConfirmation">Shipping confirmation to be inserted</param>
-        /// <param name="connSting">Connection string to SQL Server instance</param>
-        public static void InsertShippingConfirmation(ShippingConfirmationModel shippingConfirmation, string connSting)
+        /// <param name="connectionString">Connection string to SQL Server instance</param>
+        /// <exception cref="Exception">All operations around this method are surrounded by a transaction which rollbacked if an exception is encountered, commited otherwise.</exception>
```

Comment:
Where is this exception being thrown? Your comment indicates that this method throws the generate `Exception` type but it is not thrown in this method. You only want to document the exceptions that your functionality throws.